api/src/controllers/login.js

```diff
@@ -13,6 +13,7 @@ const ONE_YEAR = 31536000000;
 const logger = require('../logger');
 const db = require('../db');
 const production = process.env.NODE_ENV === 'production';
+const _locale = require('locale');
```

Comment:
Nit: rename this constant something else. We use `_` to indicate something that's exported but private. Obviously the name `locale` is overused, but what about calling it `requestLocale` or something?